landing_page/templates/landing_page/movie.html

```diff
@@ -1,31 +1,52 @@
-{% extends 'landing_page/base.html' %} {% load static %} {% block content %}
+{% extends 'landing_page/base.html' %} 
+{% load static %} 
+{% block content %}
 <link rel="stylesheet" href="{% static 'landing_page/css/movie.css' %}" />
 <script src="{% static 'landing_page/js/movie.js' %}"></script>
 
 <div id="message_div">
-    <h1>More about the Movie</h1>
+  <h1>More about the Movie</h1>
```

Comment:
i think we don't even need this header yeah? like it would look better like this:
![image](https://github.com/jonathanhouge/ReeldIn/assets/111781413/ced21cd4-235e-4e9f-b332-3b4d8fd566ee)

Reply:
i think the "more about the movie" header should be removed.